commit 1ca43c80d80f0e2e6f4f3942ab2e024104c64586
Author: Jesse Brandeburg <jesse.brandeburg@intel.com>
Date:   Thu Aug 9 06:29:01 2018 -0700

    ice: Fix potential return of uninitialized value
    
    [ Upstream commit dab0588fb616c1774bbf108eab1749dda4ac6942 ]
    
    In ice_vsi_setup_[tx|rx]_rings, err is uninitialized which can result in
    a garbage value return to the caller. Fix that.
    
    Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
    Signed-off-by: Anirudh Venkataramanan <anirudh.venkataramanan@intel.com>
    Tested-by: Tony Brelinski <tonyx.brelinski@intel.com>
    Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/ethernet/intel/ice/ice_main.c b/drivers/net/ethernet/intel/ice/ice_main.c
index aaaa1e2a7df6..27c9aa31b248 100644
--- a/drivers/net/ethernet/intel/ice/ice_main.c
+++ b/drivers/net/ethernet/intel/ice/ice_main.c
@@ -4888,7 +4888,7 @@ int ice_down(struct ice_vsi *vsi)
  */
 static int ice_vsi_setup_tx_rings(struct ice_vsi *vsi)
 {
-	int i, err;
+	int i, err = 0;
 
 	if (!vsi->num_txq) {
 		dev_err(&vsi->back->pdev->dev, "VSI %d has 0 Tx queues\n",
@@ -4913,7 +4913,7 @@ static int ice_vsi_setup_tx_rings(struct ice_vsi *vsi)
  */
 static int ice_vsi_setup_rx_rings(struct ice_vsi *vsi)
 {
-	int i, err;
+	int i, err = 0;
 
 	if (!vsi->num_rxq) {
 		dev_err(&vsi->back->pdev->dev, "VSI %d has 0 Rx queues\n",
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1291_1ca43c80d80f0e2e6f4f3942ab2e024104c64586.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1291_1ca43c80d80f0e2e6f4f3942ab2e024104c64586.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
